server/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "resume-wizard-server",
   "version": "1.0.0",
   "description": "Server for the best Resume Wizard app ever",
+  "type": "module",
```

Comment:
Nah this is to set the entire project as an `ESModules` project, which this isn't...
It's complaining when you're using `import` in that file because you're not `exporting` anything, and hence it doesn't think it's a module - it will eventually need to be a module as we won't be executing it from the `cli` in prod - just revert it to using your original `require` and when we eventually convert it to a module, we can use `import`'s then

Also I noticed you renamed `server/.eslintrc.js` and `server/jest.config.js` to `.cjs` extensions - I'm assuming this was because you added this `"type": "module"` here - can we revert that back too?

Reply:
,,|,, fixed